examples/projects/VS2015/autobahn-cpp-examples.vs2015.sln

```diff
@@ -0,0 +1,84 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
How should these VS project files be maintained by non-Windows developers, and why is the cross-platform CMake build not sufficient to generate these without committing them?